content/purgatorio/collect-and-prepare-data/data-collection.md

```diff
@@ -0,0 +1,209 @@
+---
+title: Data Collection
+author: neomatrix369
+description: The purpose of this guide is to talk about data collection and the different steps you need to do to prepare your dataset even before we can think of Machine Learning models and the works.
+---
+
+# What you will learn 
+The real world is full of data, there are many types of data and collecting them is an art and comes through experience and doing it in an iterative process.
+
+Also this is one of the preliminary steps when you are about to do Machine Learning or Deep Learning or any of the AI things. The cascading steps or processes depend quite a bit on this task but at the same time, this is a task that can be iterated a number of times.
+
+And revisited depending on what we conclude or find in the subsequent steps and processes (you can look at the index to see what these steps are). 
+
+We will also introduce a flow diagram eventually to show you where in the flow this task/step sits.
+
+The purpose of this guide is to show you the importance of these steps, but also how important data collection can be in the whole cycle.
+
+# Index
+- [How to use this guide?](#How-to-use-this-guide)
+- [The end-to-end process](#The-end-to-end-process)
+- [Research or business questions](#Research-or-business-questions)
+- [Start Small](#Start-small)
+- [Small steps](#Small-steps)
+- [The Toolkit and sources](#The-Toolkit-and-sources)
+- [Datasets](#Datasets)
+- [Data Generation](#Data-Generation)
+- [Visualisation](#Visualisation)
+- [Sanity Check](#Sanity-Check)
+  - [Ethics](#Ethics)
+  - [Interpretability / Explainability](#interpretability--explainability)
+- [Automate These Boring Stuffs!](#Automate-these-boring-stuffs)
+- [Resources](#Resources)
+- [Conclusions](#Conclusions)
+
+**Let's Start!**
+
+### How to use this guide?
+
+We'll give you some general tips to learn effectively and develop a decent foundation, that you can rely on to address and solve Data Science problems in the complexity of the real world (which is messy by definition).
+
+In this guide, you'll find a number of different tips and the sub-topics answer these questions provided below:
+- why? (see [Research or business questions](#Research-or-business-questions))
+- how?
+- what?
+
+The last two "how?" and "what?" overlap a bit and expand into these:
+- [Start Small](#Start-small)
+- [Small steps](#Small-steps)
+- [The Toolkit and sources](#The-Toolkit-and-sources)
+- [Datasets](#Datasets)
+- [Data Generation](#Data-Generation)
+- [Visualisation](#Visualisation)
+- [Sanity Check](#Sanity-Check)
+
+And to keep track of the "big picture", refer to [The end-to-end process](#The-end-to-end-process) section. Any other section(s) not mentioned are supporting sections to this guide.
+
+As stated in the Virgilio's Teaching Strategy Guide, read more about it [here](https://github.com/virgili0/Virgilio/blob/708bd547c0bd04b05ac76b25c42b0158ea632b21/content/paradiso/virgilio-teaching-strategy.md).
```

Comment:
link not working. Coming from the older Virgilio.. Try to fix it linking from the actual content folder, it should be easy with relative github links

https://stackoverflow.com/questions/7653483/github-relative-link-in-markdown-file#:~:text=Starting%20today%2C%20GitHub%20supports%20relative,using%20a%20different%20markup%20renderer.

Reply:
Yeah I thought it was strange, it found the link but then when you click it it wont work!

Thanks I'll do that.

Reply:
Done